src/Auth/helpers.js

```diff
@@ -44,7 +42,18 @@ async function blockValue(field){
     }
   }
 
+function getOwnerid(profileObject){
+  if (typeof profileObject !== "undefined"){
+      if (propertyExists(profileObject, "profile")){
+      return profileObject.profile.gcID;
+      }
+  } else {
+      return null;
+  }
```

Comment:
There is a missing return path.  For example if the profileObject is defined it falls to `propertyExists(profileObject, "profile"` but if that property does not exist the function returns no value as it skips to line 53.  The return null is never reached and the outcome is that the variable that is populated by this function is now `undefined`.